.github/workflows/ci.yaml

```diff
@@ -1,29 +1,66 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
 name: CI
+
 on:
-  push:
   pull_request:
+  push:
+    branches:
+      - master
+
 jobs:
   tests:
     runs-on: ${{ matrix.operating-system }}
+
     strategy:
       matrix:
-        operating-system: [ubuntu-latest, windows-latest, macOS-latest]
-        php-version: ['7.1', '7.2', '7.3', '7.4', 8.0']
-    name: PHP ${{ matrix.php-version }} Test on ${{ matrix.operating-system }}
+        operating-system: [ ubuntu-latest ]
+        php-version: ['7.4']
+        dependencies: ['']
+        include:
+          - { operating-system: 'ubuntu-latest', php-version: '7.1', dependencies: '--prefer-lowest' }
```

Comment:
This is testing way less than what is there now, mac is missing and windows only has 1 version. We could double the existing tests with the prefer lowest flag ?

Reply:
IMHO we do not need to test on Mac as it waste of time. Testing on linux on 100% covers Mac.
Also I've never seen that anybody in php community doing this. 

we just need to check that application can is working  find on windows. All other cases will be covered on linux.